src/apprt/gtk/Window.zig

```diff
@@ -457,12 +457,6 @@ pub fn syncAppearance(self: *Window) !void {
     self.winproto.syncAppearance() catch |err| {
         log.warn("failed to sync winproto appearance error={}", .{err});
     };
-
-    toggleCssClass(
-        @ptrCast(self.window),
-        "background",
-        self.config.background_opacity >= 1,
-    );
```

Comment:
Just as a note to reviewers, this was a duplicate call to the same toggle we do on line 422